forum/.htaccess

```diff
@@ -11,3 +11,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteCond %{REQUEST_FILENAME} !-d
 RewriteRule ^.*$ index.php?qa-rewrite=$0&%{QUERY_STRING} [L]
 </IfModule>
+
+<IfModule mod_headers.c>
+Header set X-Frame-Options "sameorigin"
+Header set X-XSS-Protection "1; mode=block"
+Header set X-Content-Type-Options "nosniff"
+Header set Strict-Transport-Security "max-age=31536000"
+Header set Content-Security-Policy "default-src 'none'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self';"

```

Comment:
Taka polityka skutecznie wytnie fonty z Google Fonts, GA, widgety społecznościowe czy reklamy. Należałoby ją odpowiednio przystosować do realnego użycia forum.

Dodatkowo warto także określić politykę dla ramek, co zezwoli choćby na osadzanie filmików z YouTube.

Dodatkowo forum ma pełno skryptów inline, które również zostaną wycięte przez obecny kształt polityki.

Reply:
Tak, wiem - wyszło to przez to, że nie dałem flagi "Work In progress" w tym PR. 
jestem w trakcie ustawiania tego. Miło będzie jeśli sprawdzisz gdy będzie to już wg mnie dobrze ustalone. Jestem świezy w temacie więc mogę coś nieumyślnie źle zrobić